src/server/qgswfsserver.cpp

```diff
@@ -445,7 +445,7 @@ int QgsWFSServer::getFeature( QgsRequestHandler& request, const QString& format
     QDomElement queryElem;
     for ( int i = 0; i < queryNodes.size(); i++ )
     {
-      queryElem = queryNodes.at( 0 ).toElement();
+      queryElem = queryNodes.at( i ).toElement();
```

Comment:
Travis is still failing for an indentation error in the Python code, would you please fix it so that we can finally merge?
See the error at:
https://travis-ci.org/qgis/QGIS/jobs/260609197#L1202

Thanks for your patience.